test/new-e2e/pkg/runner/profile.go

```diff
@@ -137,3 +200,21 @@ func GetProfile() Profile {
 
 	return runProfile
 }
+
+// GetTestOutputDir returns the output directory for a specific test.
+// The test name is sanitized to remove invalid characters, and the output directory is created.
+func GetTestOutputDir(p Profile, t *testing.T) (string, error) {
+	// https://en.wikipedia.org/wiki/Filename#Reserved_characters_and_words
+	invalidPathChars := strings.Join([]string{"?", "%", "*", ":", "|", "\"", "<", ">", ".", ",", ";", "="}, "")
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Don't we have shared code to sanitise directory names ?

Reply:
not that I'm aware of